docs/lib/predictions/fragments/android/getting-started.md

```diff
@@ -0,0 +1,171 @@
+
+When your backend is successfully updated, verify that `amplifyconfiguration.json` and `awsconfiguration.json` are properly updated inside the `res/raw` directory. Your new configuration file `amplifyconfiguration.json` will contain your default project `region` value as well as any necessary configuration options for each predictions method (i.e. target language and source language for translate, etc).
+
+
+## Manual Configuration
+
+If you have already created the resources in the cloud and would like to take advantage of those existing resources but still use the Amplify library in Android Studio, please follow the directions below:
+
+Create the file `amplifyconfiguration.json`:
+```
+touch amplifyconfiguration.json
+```
+
+Copy the contents over and update the values for the specific predictions method you are looking to use:
+```
+{
+    "UserAgent": "aws-amplify-cli/2.0",
+    "Version": "1.0",
+    "Predictions": {
+        "plugins": {
+            "awsPredictionsPlugin": {
+                "defaultRegion": "us-west-2",
+                 "identify": {
+                    "identifyText": {
+                        "format": "ALL",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "identifyEntities": {
+                        "maxEntities": "0",
+                        "celebrityDetectionEnabled": "true",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "identifyLabels": {
+                        "region": "us-west-2",
+                        "type": "LABELS",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "convert": {
+                    "translateText": {
+                        "targetLang": "zh",
+                        "sourceLang": "en",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "speechGenerator": {
+                        "voice": "Salli",
+                        "language": "en-US",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "interpret": {
+                    "interpretText": {
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+Add both the `amplifyconfiguration.json` and the `awsconfiguration.json` to your project inside the `res/raw` directory.
+
+## Configure your application
+
+Add the following dependencies to your **app**'s' `build.gradle` file and click "Sync Now" when prompted:
+
+```gradle
+dependencies {
+    implementation 'com.amplifyframework:core:0.10.0'
```

Comment:
`1.0.0`

Reply:
oof you are right. It's not added in this version yet haha